src/index.html

```diff
@@ -28,101 +28,101 @@ <h2>Filter</h2>
         <div class="filters">
           <div class="single-filter">
             <label for="antipasti">
-              <input type="checkbox" id="antipasti" name="antipasti" value="yes">Antipasti
+              <input class="filter-checkbox tag" type="checkbox" id="antipasti" value="yes">Antipasti
```

Comment:
I'm thinking we can get rid of the type="checkbox" thing since we're technically not utilizing it. Might hold off a bit just in case.

Reply:
Lexy mentioned that we ended up not utilizing this, I haven't caught up to that point yet in the code reviews, but I really think that is the direction we need to go with these. I think it will hurt us on our score if we are using an input element in our html but are not actually capturing the input and rather just using it as a button. I will do some research into checkboxes and hopefully get a better understanding of how to best utilize them!